__tests__/fixtures/upgrade-interactive/up-to-date/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "yarn-test",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "left-pad": "^1.1.3"
```

Comment:
Since this is an external dep, it may break in the future but I think it is okay for a while? :D

Reply:
I assumed it would use the mocked request to fetch the left-pad metadata, in which 1.1.3 is the greatest.

Reply:
Oh, right. That's probably true.

Reply:
Well I'm glad your confidence level rivals my own 😆